### PR TITLE
Fix Search Results List in Reflectometry not resizing

### LINKED ISSUE #27177: Search results list no longer expands to fill space
### Expected behavior
The search results list should expand to fill all available space

### Actual behavior
There is unused space above the search results list.

### Steps to reproduce the behavior
- Open the ISIS Reflectometry GUI
- Resize the window to expand vertically
refl_gui_layout_broken

### Platforms affected
Ubuntu 18.04, probably all

### PR DESCRIPTION
**Pull request by DavidFair**
**Description of work.**
Fixes the search result list in reflectometry not resizing correctly.
Previously it would resize the other elements instead of the results
list.

**To test:**
- Open the Reflectometry GUI
- Drag the box down, check that search results resizes (see original issue)

Fixes #27177. 

*This does not require release notes* because it is a layout fix

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by RichardWaiteSTFC — DISMISSED** [review]
Search list now resizes with window - bug has been fixed.

File: qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsWidget.ui [hunk 1]
```diff
@@ -74,7 +66,7 @@
          <item row="0" column="0">
           <widget class="QLabel" name="labelInstrument">
            <property name="text">
-            <string>&amp;Instrument:</string>
+            <string>Instru&amp;ment:</string>
```
Comment (DanNixon) [inline qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsWidget.ui]: Why was this changed?

**Review by DanNixon — APPROVED** [review]
